lib/array.js

```diff
@@ -59,8 +59,13 @@ const asyncMap = (items, fn, options = {}, done) => {
     options = DEFAULT_OPTIONS;
   }
 
-  if (!items.length) {
-    if (done) done(null, []);
+  const len = items.length || items.size;
```

Comment:
`asyncMap` can be replaced with `iter().map().throttle()`

Reply:
What do you mean? Rewrite the whole `asyncMap` using `throttle`? or drop `asyncMap`?

Reply:
@KHYehor yes, you should delete current implementation and replace it with `map().throttle()`.